src/deploy/NVA_build/NooBaa.Dockerfile

```diff
@@ -60,6 +60,7 @@ ENV ENDPOINT_NODE_OPTIONS ''
 
 RUN dnf install -y -q bash \
     lsof \
+    procps \
```

Comment:
in Centos8 procps looks like an alias for procps-ng
We already have the package in the NooBaa image:
```
$ docker run -it --name delete --rm noobaa bash
bash-4.4$ rpm -qa | grep procps
procps-ng-3.3.15-1.el8.x86_64
```

Reply:
@liranmauda so why was it missing on the endpoint?